recipes/qemu-user-execve/patches/0001-intercept-qexecve.patch

```diff
@@ -0,0 +1,25 @@
+--- qemu/linux-user/main.c
```

Comment:
What is the source of these patches?

Reply:
Yes, if you have not written them by yourself, they should have the initial authors attached to them. Still, if you had written them, they should reference you as the author.

Reply:
Got it.

So, the purpose of this recipe was mostly to resolve the ZIG cross-build. This patch was never accepted by the QEMU dev team. They argued that proot and binfmt_misc already served that purpose, but I find that it is simpler to use qemu+execve to resolve the ZIG build - Just want to make sure you know the full context
I only proposed this recipe after you suggested that it could be useful